content/en/docs/tasks/debug-application-cluster/local-debugging.md

```diff
@@ -2,60 +2,67 @@
 title: Developing and debugging services locally
 content_type: task
 ---
-
+ 
 <!-- overview -->
-
-Kubernetes applications usually consist of multiple, separate services, each running in its own container. Developing and debugging these services on a remote Kubernetes cluster can be cumbersome, requiring you to [get a shell on a running container](/docs/tasks/debug-application-cluster/get-shell-running-container/) and running your tools inside the remote shell.
-
-`telepresence` is a tool to ease the process of developing and debugging services locally, while proxying the service to a remote Kubernetes cluster. Using `telepresence` allows you to use custom tools, such as a debugger and IDE, for a local service and provides the service full access to ConfigMap, secrets, and the services running on the remote cluster.
-
+ 
+Kubernetes applications usually consist of multiple, separate services, each running in its own container. Developing and debugging these services on a remote Kubernetes cluster can be cumbersome, requiring you to [get a shell on a running container](https://kubernetes.io/docs/tasks/debug-application-cluster/get-shell-running-container/) in order to run your tools inside that remote shell.
```

Comment:
```suggestion
Kubernetes applications usually consist of multiple, separate services, each running in its own container. Developing and debugging these services on a remote Kubernetes cluster can be cumbersome, requiring you to [get a shell on a running container](/docs/tasks/debug-application-cluster/get-shell-running-container/) in order to run your tools inside that remote shell.
```

We use reference style links with Hugo instead of direct links everywhere, could you please revert this? 🙂

Reply:
+1 to this

Reply:
Thanks for the feedback, @RinkiyaKeDad & @divya-mohan0209. I just pushed a fix for this.